hypha/apply/api/v1/serializers.py

```diff
@@ -199,9 +204,62 @@ def get_meta_questions(self, obj):
         ]
         return data
 
+    def get_screening(self, obj):
+        selected_default = {}
+        selected_reasons = []
+        all_screening = []
+
+        for screening in obj.screening_statuses.values():
+            if screening['default']:
+                selected_default = screening
+            else:
+                selected_reasons.append(screening)
+
+        for screening in ScreeningStatus.objects.values():
+            all_screening.append(screening)
+
+        screening = [
+            {
+                "selected_reasons": selected_reasons
+            },
+            {
+                "selected_default": selected_default
+            },
+            {
+                "all_screening": all_screening
+            }
+        ]
+        return screening
+
+    def get_flags(self, obj):
+        flags = [
+            {
+                "type": 'user',
+                "selected": obj.flagged_by(self.context['request'].user)
+            },
+            {
+                "type": 'staff',
+                "selected": obj.flagged_staff
+            }
+        ]
+
+        return flags
+
     def get_questions(self, obj):
         return self.serialize_questions(obj, obj.normal_blocks)
 
+    def get_reminders(self, obj):
+        reminders = []
+        for reminder in obj.reminders.all():
+            reminders.append({
+                "id": reminder.id,
+                "submission_id": reminder.submission_id,
+                "title": reminder.title,
+                "action_type": reminder.action_type,
+                "is_expired": reminder.is_expired
+            })
+        return reminders
```

Comment:
While I'm only supposed to review React code, a suggestion for future consideration:

(No need to address this).

This can be replaced with a [`.values()`](https://docs.djangoproject.com/en/3.2/ref/models/querysets/#values) call:

```suggestion
    def get_reminders(self, obj):
        return obj.reminders.values('id', 'submission_id', 'title', 'action_type', 'is_expired')
```

Reply:
using `.values()`, won't allow me to access `action_type` which is a property defined in the model

Reply:
Ah! makes sense. Thanks for clarifying.